### PR TITLE
Route dev

### PR DESCRIPTION
**Pull request by hc09141**
I've created a KPSRouteController with some basic views. Updating the route only allows a change in weight price and volume price. Updating also creates an update price event.

### CODE REVIEW

File: app/controllers/kps_routes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    @kps_route = KpsRoute.find_by_id(params[:id])
+    if @kps_route.update(kps_route_price_update_params)
+      UpdatePrice.create(
+        origin: @kps_route.origin,
+        destination: @kps_route.destination,
+        priority: @kps_route.priority,
+        revenue: 0,
+        expenditure: 0,
+        weight_price: @kps_route.weight_price,
+        volume_price: @kps_route.volume_price)
```
Comment (TanaTanoi) [inline app/controllers/kps_routes_controller.rb]: new line before the bracket so the `#create` forms a nice block

File: app/controllers/kps_routes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      redirect_to @kps_route
+    end
+  end
+
+  private
```
Comment (TanaTanoi) [inline app/controllers/kps_routes_controller.rb]: new line after private; rule of thumb is to separate stuff on the same indentation level

File: app/views/kps_routes/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<h1>Update cost of KPS Route</h1>
+<p>
+  <%= "Origin: #{@kps_route.origin.city}, #{@kps_route.origin.country}" %>
+  <%= "Destination: #{@kps_route.destination.city}, #{@kps_route.destination.country}" %>
+  <%= "Priority: #{@kps_route.priority}" %>
+</p>
+
+<%= form_for @kps_route do |f| %>
+  <label>Weight price:</label>
+  <%= f.text_field :weight_price, placeholder: "#{number_with_precision(@kps_route.weight_price, precision: 2)}" %>
```
Comment (TanaTanoi) [inline app/views/kps_routes/edit.html.erb]: I _think_ you don't need the string interpolation for this, so you can just get rid of the `"#{` and `}"`. Same with below

Comment (hc09141) [inline app/views/kps_routes/edit.html.erb]: Tested this and it still works without the #{ and }

**Pull request merged**